### ISSUE TITLE
"Metadata KeyValues does not exist" in ImageClassification

### ISSUE THREAD

**Issue by jdahan91** [dotnet/machinelearning-modelbuilder#796]
Choosing image classification, i'm trying to train a folder with three subfolders each subfolder has one image.

But I don't understand what's the problem that gives me this error. I tried including a .tsv files but still doesn't help.

**Comment by beccamc** [comment]
@jdahan91 sorry for the long delay in helping with this issue. I can't see the training data you're trying to use, but only one image for each label wouldn't be enough data. This is failing because it's trying to break your dataset into a training and test set. It can't do that with only one image. 

Try increasing the number of images - I would provide at least 10 - and see if it works.  Let me know how it goes!

**Comment by jdahan91** [comment]
> @jdahan91 sorry for the long delay in helping with this issue. I can't see the training data you're trying to use, but only one image for each label wouldn't be enough data. This is failing because it's trying to break your dataset into a training and test set. It can't do that with only one image.
> 
> Try increasing the number of images - I would provide at least 10 - and see if it works. Let me know how it goes!

Thank you for the reply, I eventually went with a different approach in the app I wanted to use machine learning with, but I will remember this. Thanks, if there's any updates on my side i'll update here.

**Comment by beccamc** [comment]
We should update to show an error message if there are less than 5 images for each class.

**Comment by beccamc** [comment]
@zewditu Validating your change, it looks like we're still enabling the next step when users don't have enough images. We know this will fail so we shouldn't allow people to continue without fixing it. 

![image.png](https://images.zenhubusercontent.com/5d55c3d68fac840001314eaa/74ace7db-615d-4567-96c0-da1478070ba0)

**Comment by beccamc** [comment]
This is fixed 
![image.png](https://images.zenhubusercontent.com/5d55c3d68fac840001314eaa/b30838db-57f8-4768-8a0f-bfe54d1a7683)
